src/app/components/job/action/add/job-preparation-task-picker.html

```diff
@@ -20,7 +20,7 @@
             <bl-error controlName="maxTaskRetryCount" code="validateRange">Retry count values can range from -1 to 100</bl-error>
         </bl-form-field>
         <bl-form-field>
-            <bl-duration-picker formControlName="retentionTime" label="Retention time"></bl-duration-picker>
+            <bl-duration-picker formControlName="retentionTime" label="Retention time" [required]="true"[allowUnlimited]="false" [defaultDuration]="7"></bl-duration-picker>
```

Comment:
is that all the bl-duration-picker uses? would it make sense to remove unlimited option all together
?

Reply:
Its used for maxwallclocktime as well which makes use of unlimited as a value.